src/client/pydaos/pydaos_shim.c

```diff
@@ -371,8 +370,7 @@ __shim_handle__cont_newobj(PyObject *self, PyObject *args)
 	if (hdl->alloc.hi >= MAX_OID_HI) {
 		rc = daos_cont_alloc_oids(hdl->coh, 1, &hdl->alloc.lo, NULL);
 		if (rc) {
-			D_ERROR("daos_cont_alloc_oids() Failed "DF_RC"\n",
-				DP_RC(rc));
+			D_ERROR("daos_cont_alloc_oids() Failed "DF_RC"\n", DP_RC(rc));
```

Comment:
Do you want this to have a `:` like the others?
```suggestion
			D_ERROR("daos_cont_alloc_oids() Failed: "DF_RC"\n", DP_RC(rc));
```

Reply:
yes. not sure how your eyes caught that :-)

Reply:
Much time spent as a teaching assistant grading papers and catching mistakes :)